### DISCUSSION TITLE
第6节课作业(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#415] [作业]
请在下方提交作业链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course6_hw.md

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135712271

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/homework5_opencompass.md

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/homework06.md

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/6homework.md

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E5%85%AD%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/homework6/blob/main/homework6.md

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/HWDay06.md

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by Alias-z** [comment]
388+https://github.com/Alias-z/InternLM-Camp-2024/tree/main/Assignment%205%20OpenCompass
Note：如果你的预测结果一直有问题，找不到解决办法的话，试试git上的opencompass，和gitee上的完全是两个版本。4小时的血泪 :(

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson6/assign.md

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/00121717-f251-4d48-81cc-c71dd8f42754

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135780991

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9386241

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/PZBfdgwyto0NTixGeCNcrDRunGd?from=from_copylink
